src/util/util.ts

```diff
@@ -178,6 +180,56 @@ export function convertLootBanksToItemBanks(lootResult: LootBank): Record<string
 export function getAncientShardChanceFromHP(hitpoints: number): number {
 	return Math.round((500 - hitpoints) / 3);
 }
+
 export function getTotemChanceFromHP(hitpoints: number): number {
 	return 500 - hitpoints;
 }
+
+export interface RevTable {
+	uniqueTable: RevTableItem;
+	ancientEmblem: RevTableItem;
+	ancientTotem: RevTableItem;
+	ancientCrystal: RevTableItem;
+	ancientStatuette: RevTableItem;
+	topThree: RevTableItem;
+	seeds: RevTableItem;
+}
+
+type RevTableItem = [number, number];
+
+export const revsUniqueTable = new LootTable()
+	.add('Amulet of avarice', 1, 2)
+	.add("Craw's bow (u)", 1, 1)
+	.add("Thammaron's sceptre (u)", 1, 1)
+	.add("Viggora's chainmace (u)", 1, 1);
+
+export function makeRevTable(table: RevTable): CustomKillLogic {
+	return (options: MonsterKillOptions, currentLoot: Bank) => {
+		const index = options.skulled ? 1 : 0;
+		if (roll(table.uniqueTable[index])) {
```

Comment:
Each of these rolls are independant, meaning not only is it possible to get 1) a unique, 2) yew seed, 3) magic seed, BUT you  can also get 1 of each emblem, theoretically from 1 kill.

While the chances of this are low, because the rolls are independant, it also **dramatically** increases the chance to get any of the items, because you're rolling multiple times.

This function should return after a successful roll, rolling in order from uniques => emblems => seeds, in my opinion. This slightly increases the chance of getting a successful roll, but is how Jagex has been drop tables lately.